src/bindings/python/package/__init__.py

```diff
@@ -16,10 +29,11 @@
 # environment variable to 0.
 #
 
-if sys.version_info >= (3, 8) and platform.system() == "Windows" and os.getenv("OCIO_PYTHON_LOAD_DLLS_FROM_PATH", "1") == "1":
-    for path in os.getenv("PATH", "").split(os.pathsep):
-        if os.path.exists(path) and path != ".":
-            os.add_dll_directory(path)
+if sys.version_info >= (3, 8) and platform.system() == "Windows":
+    if os.getenv("OCIO_PYTHON_LOAD_DLLS_FROM_PATH", "1") == "1":
```

Comment:
Unrelated to this PR (just a general comment): it would be better if this was off by default (i.e. opt-in, not opt-out).

Reply:
I generally agree with that although I didn't investigate enough yet the pros and cons, I think we should decide the desired behaviour in the corresponding issue https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1785. When that's settle I can make the change, probably in a new PR to not block this one.